data/hai/hai culture conversations.txt

```diff
@@ -329,3 +329,57 @@ mission "Hai Film Release"
 				decline
 			label translator
 			`	You find a movie theater showing Likeet's film, but your translator is of little help here. The movie is sparse on dialog, and the few lines spoken in the movie seem to be cryptic references to Hai literature and philosophy. The most you are able to make out is that the movie is about an unseen monster inhabiting the station that slowly drives those it meets insane. It seems to be a hit with the Hai audience, though, and you leave the theater feeling distinctly unnerved.`
+
+
+
+mission "Sick of Acorn Bread"
+	minor
+	source
+		government "Hai"
+		not attributes "station"
+	to offer
+		credits > 50
+		random < 1
+	on offer
+		conversation
+			`With some time to kill before your next mission, you enter the spaceport searching for a bite to eat. Restaurants advertise their menus in bright colors on their windows, each competing for your attention. You spot one that promises "authentic Hai cuisine," along with some large text printed in the Hai language which you don't understand. Next to it, you see a human street food vendor offering snacks and ready-made meals.`
```

Comment:
What if you have the translator? Does it just not work on written text? (It could, I'm not sure)